starknet_py/tests/e2e/conftest.py

```diff
@@ -356,24 +359,24 @@ def map_contract(request) -> Contract:
 
 
 @pytest.fixture(scope="module")
-def map_source_code() -> str:
+def map_source_code() -> CairoSourceCode:
     """
     Returns source code of the map contract
     """
-    return (contracts_dir / "map.cairo").read_text("utf-8")
+    return cast(CairoSourceCode, (contracts_dir / "map.cairo").read_text("utf-8"))
 
 
 @pytest.fixture(scope="module")
-def erc20_source_code() -> str:
+def erc20_source_code() -> CairoSourceCode:
     """
     Returns source code of the erc20 contract
     """
-    return (contracts_dir / "erc20.cairo").read_text("utf-8")
+    return cast(CairoSourceCode, (contracts_dir / "erc20.cairo").read_text("utf-8"))
```

Comment:
Same as I wrote in #404 Ideally there should be a solution to make pyright work with these `NewType`s without type casting.